store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreTestUtil.java

```diff
@@ -0,0 +1,189 @@
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.gchq.gaffer.user.StoreUser.ALL_USERS;
+import static uk.gov.gchq.gaffer.user.StoreUser.authUser;
+import static uk.gov.gchq.gaffer.user.StoreUser.blankUser;
+import static uk.gov.gchq.gaffer.user.StoreUser.testUser;
+import static uk.gov.gchq.gaffer.user.User.UNKNOWN_USER_ID;
+
+public final class FederatedStoreTestUtil {
+
+    public static final String STRING = "String";
+    public static final String ENTITIES = "Entities";
+    public static final String EDGES = "Edges";
+    public static final String GRAPH_ID_TEST_FEDERATED_STORE = "testFederatedStoreGraphID";
+    public static final String GRAPH_ID_ACCUMULO = "AccumuloStore";
+    public static final String GRAPH_ID_MAP = "MapStore";
+    public static final String ACCUMULO_STORE_SINGLE_USE_PROPERTIES = "properties/singleUseAccumuloStore.properties";
+    public static final String MAP_STORE_SINGLE_USE_PROPERTIES = "properties/singleUseMapStore.properties";
+    public static final String GRAPH_ID_ACCUMULO_WITH_EDGES = GRAPH_ID_ACCUMULO + EDGES;
+    public static final String GRAPH_ID_ACCUMULO_WITH_ENTITIES = GRAPH_ID_ACCUMULO + ENTITIES;
+
+    public static final String GRAPH_ID_A = "graphA";
+    public static final String GRAPH_ID_B = "graphB";
+    public static final String FEDERATED_STORE_SINGLE_USE_PROPERTIES = "properties/singleUseFederatedStore.properties";
+    public static final String SCHEMA_EDGE_BASIC_JSON = "/schema/basicEdgeSchema.json";
+    public static final String SCHEMA_ENTITY_BASIC_JSON = "/schema/basicEntitySchema.json";
+    public static final String SCHEMA_ENTITY_A_JSON = "/schema/entityASchema.json";
+    public static final String SCHEMA_ENTITY_B_JSON = "/schema/entityBSchema.json";
+    public static final String GROUP_BASIC_EDGE = "BasicEdge";
+    public static final String GROUP_BASIC_ENTITY = "BasicEntity";
+    public static final String BASIC_VERTEX = "basicVertex";
+    public static final String FORMAT_PROPERTY_STRING = "property%s";
+    public static final String PROPERTY_1 = property(1);
+    public static final String PROPERTY_2 = property(2);
+    public static final int VALUE_PROPERTY1 = 1;
+    public static final String SOURCE_BASIC = "basicSource";
+    public static final String DEST_BASIC = "basicDest";
+    public static final String ACCUMULO_STORE_SINGLE_USE_PROPERTIES_ALT = "properties/singleUseAccumuloStoreAlt.properties";
+    public static final String FORMAT_VALUE_STRING = "value%s";
+    public static final String VALUE_1 = value(1);
+    public static final String VALUE_2 = value(2);
+    static final String CACHE_SERVICE_CLASS_STRING = "uk.gov.gchq.gaffer.cache.impl.HashMapCacheService";
+    static final Set<String> GRAPH_AUTHS_ALL_USERS = ImmutableSet.of(ALL_USERS);
+    public static final String INTEGER = "integer";
+
+    private FederatedStoreTestUtil() {
+        //no instance
+    }
```

Comment:
Is this needed?

Reply:
The utility class is blocked from having an instance created, the private constructor is not accessible.
comment shows its deliberate choice that is empty.